src/api/enums/BidRejectReason.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-2019, The Particl Market developers
+// Distributed under the GPL software license, see the accompanying
+// file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE
+
+/**
+ * BidRejectReason
+ *
+ */
+
+export enum BidRejectReason {
+    OUT_OF_STOCK = 'OUT_OF_STOCK'
```

Comment:
should we add NO_REASON or something to be used as a default when no specific reason wasnt given?